app/services/random_content.js

```diff
@@ -2,8 +2,7 @@ const axios = require('axios');
 const logger = require('../logger');
 const errors = require('../errors');
 const errorsCatalog = require('../../app/schemas/errors_catalog');
-
-const urlRandomContent = 'https://geek-jokes.sameerkumar.website/api?format=json';
+const { urlRandomContent } = require('../../config').common.session;
```

Comment:
Its not relation with session.
You should create a new property en common to reference the joke api:
```js
    jokes-api: {
      url: 'https://geek-jokes.sameerkumar.website'
    }
```
And then, in the request:

```.js
await axios.request({
  method: 'get',
  url: `${url}/api?format=json`,
})

```